### DISCUSSION TITLE
Change git pull command?

### DISCUSSION THREAD

**Discussion by iammati** [deployphp/deployer#2938] [General]
Hi there,

I'm developing at the moment an UI based on Vue.js for Deployer which works and runs pretty nice so far, just the `git pull` command is still an issue yet.

It requires the credentials to prompted by default, is there a way to change the git pull command? Like passing credentials (even though it is not recommended) into the remote server or anything similar?

Thanks

**Comment by antonmedv** [comment]
My recommendation: use deploy keys.

**Reply by iammati** [reply]
OK thanks
